x-pack/plugin/repositories-metering-api/src/test/java/org/elasticsearch/xpack/repositories/metering/AbstractRepositoriesMeteringAPIRestTestCase.java

```diff
@@ -256,19 +257,21 @@ private void snapshotAndRestoreIndex(String snapshot, CheckedBiConsumer<String,
 
     private void assertRequestCountersAccountedForReads(RepositoryStatsSnapshot statsSnapshot) {
         RepositoryStats repositoryStats = statsSnapshot.getRepositoryStats();
-        Map<String, Long> requestCounts = repositoryStats.requestCounts;
+        Map<String, BlobStoreActionStats> requestCounts = repositoryStats.actionStats;
```

Comment:
Nit: can we also rename the `requestCounts` here and other places in this test class to something like `actionStats`?

Reply:
Addressed in ca36b821c98f480f2f87102481f4186b73490c93